bq-metrics-extractor/script/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -x
+
+# Start app
+exec java \
+    -Dfile.encoding=UTF-8 \
+    -jar /bq-metrics-extractor.jar query --pushgateway pushgateway:8080 config/config.yaml
```

Comment:
This won't work in docker-compose, can it be amended so that it does work in docker compose (perhaps fiddling with ports in the docker-compose's config file?)